Packs/AWS-SecurityHub/Integrations/AWSSecurityHubEventCollector/AWSSecurityHubEventCollector.py

```diff
@@ -107,6 +108,10 @@ def get_events(client: "SecurityHubClient", start_time: dt.datetime | None = Non
 
         response = client.get_findings(**kwargs)
         result = response.get('Findings', [])
+
+        # Filter out events based on id_ignore_set
+        result = [event for event in result if event.get('Id') not in id_ignore_set]
```

Comment:
Not something too important, but I think this should be more appropriate:
```suggestion
        result = [event for event in result if event['Id'] not in id_ignore_set]
```
since we always expect ID in responses (and if not, we want to raise an error since something went wrong), plus we already fetch the ID using `event['id']` (and not using `get`) in other places, so it will fail anyway in such case.